.github/workflows/ci.yaml

```diff
@@ -1,7 +1,7 @@
 on:
   push:
     branches:
-      - $default-branch
```

Comment:
@sofisl after we merge this, we can update the templates in synthtool to use `$default-branch`.